Jenkinsfile

```diff
@@ -21,6 +21,7 @@ pipeline {
                     steps {
                         sh 'podman run -it --rm --pull=never localhost/$IMAGE_NAME which rstudio'
                         sh 'podman run -it --rm --pull=never localhost/$IMAGE_NAME R -q -e "getRversion() >= \\"4.1.3\\"" | tee /dev/stderr | grep -q "TRUE"'
+                        sh 'podman run -it --rm --pull=never localhost/$IMAGE_NAME python -c "import pandas, numpy, tqdm, transformers, torch"'
```

Comment:
After this import, consider running `print(torch.rand(5,3))` per the torch docs: https://pytorch.org/get-started/locally/#linux-verification

Similarly, the test for transformers is: `from transformers import pipeline; print(pipeline('sentiment-analysis')('I love you'))`

Ref. https://huggingface.co/docs/transformers/installation#install-from-source

I just tested out all the the above in a python container and they seem useful.